lib/docusign_rest/client.rb

```diff
@@ -426,6 +426,22 @@ def get_tabs(tabs, options, index)
       tab_array
     end
 
+    # Allows adding extra attributes to the file array
+    def create_file_ios_with_params(files)
+      ios = []
```

Comment:
Could there be a better name for this besides 'ios'? Same for other places, besides the modules which require that structure.